package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fullstory/data-layer-observer",
-  "version": "4.0.1",
+  "version": "4.0.2",
```

Comment:
Was `npm version` used to bump this? I would've expected `package-lock.json` to be updated as well.